src/utils/index.js

```diff
@@ -319,27 +320,32 @@ export const getLifeStageRange = memoize((
  * @param {farmhand.crop} crop
  * @returns {number}
  */
-export const getGrowingPhase = memoize(crop => {
-  const { itemId, daysWatered } = crop
-  const { cropTimetable } = itemsMap[itemId]
+export const getGrowingPhase = memoize(
+  crop => {
+    const { itemId, daysWatered } = crop
+    const { cropTimetable } = itemsMap[itemId]
 
-  if (cropTimetable[GROWING].length === 1) {
-    return 0
-  }
+    if (cropTimetable[GROWING].length === 1) {
+      return 0
+    }
 
-  // daysWatered is 0 based so we add 1 to it for this math
-  let daysGrowing = daysWatered + 1 - cropTimetable[SEED]
-  let phase = 1
+    // daysWatered is 0 based so we add 1 to it for this math
+    let daysGrowing = daysWatered + 1 - cropTimetable[SEED]
+    let phase = 1
 
-  for (let value of Object.values(cropTimetable[GROWING])) {
-    if (daysGrowing - value <= 0) break
+    for (let value of Object.values(cropTimetable[GROWING])) {
+      if (daysGrowing - value <= 0) break
 
-    daysGrowing -= value
-    phase += 1
-  }
+      daysGrowing -= value
+      phase += 1
+    }
 
-  return phase
-})
+    return phase
+  },
+  {
+    cacheSize: PURCHASEABLE_FIELD_SIZES.get(3).columns,
```

Comment:
This needs to account for every plot in the field:

```suggestion
    cacheSize: PURCHASEABLE_FIELD_SIZES.get(3).columns * PURCHASEABLE_FIELD_SIZES.get(3).rows,
```

Reply:
oops, you said that in your previous comment as well and i still did it wrong haha